.github/workflows/remainder-bot.yml

```diff
@@ -0,0 +1,77 @@
+#####################################################################################
+# Remainder Bot
+#
+# Workflow starts when:
+# 1) every day at 00:00 UTC
+#
+# Workflow:
+# 1) Check and close inactive issues and PRs
+# 2) Remind user to provide the requested feedback for issues
+# 3) Remind user to provide the requested feedback for PRs
+# 4) If no response is received in the next 15 days, we will close this issue
+# 5) If no response is received in the next 15 days, we will close this pull request
+# 6) Close the issue with a message after 60 days
+# 7) Close the pull request with a message after 60 days
+#####################################################################################
+
+name: "feedback-reminder-bot"
+
+on:
+  schedule:
+    # every day at 00:00 UTC
+    - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+concurrency:
+  group: feedback-reminder-bot
+  cancel-in-progress: true
+jobs:
+  close-inactive-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check and close inactive issues and PRs
+        uses: actions/stale@v9
+        with:
```

Comment:
Please add a start date of today to let us see how this works gong forward, once we are confident in this action, we can remove this and allow all existing PRs and issues to be affected: https://github.com/actions/stale?tab=readme-ov-file#start-date

Reply:
cc @romani ^

Reply:
If all is based labels, we don't have much issues and pulls with such labels, I will notice new comments on issues/PR for sure.

In my mind no harm to skip start date.

Reply:
Ok, let’s skip it then